packages/devtools_app/lib/src/shared/diagnostics/references.dart

```diff
@@ -87,9 +86,9 @@ Future<void> addChildReferences(
             ),
           ),
       ]);
-      break;
+      return;
```

Comment:
Nit: Are these returns necessary? We won't require `break`s in `case` statements starting with 3.0, so it'd probably be cleaner to leave them as is and then remove them when we can.

Reply:
cool!

In this exact case the method early returns sometimes in the middle of the case. So, I want it to be consistent  and make sure all options do not count on any 'after switch' operations in this method.

Reply:
Where can I read more about non-required break? Right now absence of break means  that the case is combined with next one. So, I am wondering how backward compatibility is handled.